godb/tasks/godb.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: create systemd file
+  become: true
+  notify: reload systemd-daemon
+  copy:
+    dest: /lib/systemd/system/godb.service
+    content: |
+      [Unit]
+      Description=GoDB container service
+      Requires=docker.service
+      After=docker.service
+      [Service]
+      ExecStart=docker run --rm \
+          -v /home/ubuntu/GoDB:/GoDB \
+          -v /home/ubuntu/data:/data \
+          -p 8085:8085/tcp --name godb godb
+      [Install]
+      WantedBy=multi-user.target
+
+- meta: flush_handlers
+
+- name: clone godb git repo
+  register: clone
+  git:
+    repo: https://github.com/hicresearch/GoDb
+    dest: /home/ubuntu/GoDB
+    version: deploy
```

Comment:
Do you reckon we can start tagging GoDb? Or are there still too many pending changes?